labcontrol/gui/js_tests/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
I do not understand why we are checking in this file ... isn't it a POC test that we don't want in the master codebase?  Or does the testing fail if it has nothing to work on?

Reply:
It's just there as a reference; we'd replace it with your tests. Good point though, I should verify its behavior with zero tests.

Reply:
Verified behavior is currently script will return 0 (success) when zero tests are found.